README.md

```diff
@@ -50,6 +50,83 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described below:<sup
+id="sanctuary-types-return">[1](#sanctuary-types)</sup>
+
+* `::` _"has type of"._
```

Comment:
I believe "is a member of" is preferable to "has type of". The latter implies that a value's type is a property of the value. It's better to encourage readers to think of types as sets of values. Just as the values in the set `{1,2,3}` are oblivious to their membership of the set, `true` and `false` are oblivious to their membership of `{true,false}` (which we refer to as `Boolean`).

Using "has type of" also encourages readers to think in terms of `typeof` and `instanceof`, which—ignoring inheritance—encourage us to think of every value as belonging in exactly one set of values.